library/src/main/java/com/bumptech/glide/signature/MediaStoreSignature.java

```diff
@@ -26,8 +27,8 @@
    * @param orientation  The orientation of the media store media. Ok to default to 0. See {@link
    *                     android.provider.MediaStore.Images.ImageColumns#ORIENTATION}.
    */
-  public MediaStoreSignature(String mimeType, long dateModified, int orientation) {
-    this.mimeType = mimeType;
+  public MediaStoreSignature(@Nullable String mimeType, long dateModified, int orientation) {
+    this.mimeType = mimeType == null ? "" : mimeType;
```

Comment:
Add @NonNull to the mimeType instance variable?

Reply:
SGTM.